skill-tree/src/main/resources/application.properties

```diff
@@ -8,4 +8,4 @@ jwt.rds.public.key=${RDS_PUBLIC_KEY}
 API_V1_PREFIX=/api/v1
 spring.datasource.version=8.1.0
 management.endpoints.web.exposure.include=health,info,metrics
-
+logging.level.root=DEBUG
```

Comment:
please change this for Error only for here, and debug you can use for local system..?